### PR TITLE
gulrak_filesystem, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+SOURCE_FILENAME="gulrak_filesystem-v$portVersion.tar.gz"
+SOURCE_DIR="filesystem-$portVersion"
+PATCHES="163.patch"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gulrak_filesystem$secondaryArchSuffix = $portVersion
+	devel:filesystem$secondaryArchSuffix = $portVersion
```
Comment (Begasus) [inline dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe:28]: Maybe I should use gulrak_filesystem here too?
```diff
-	devel:filesystem$secondaryArchSuffix = $portVersion
+	devel:gulrak_filesystem$secondaryArchSuffix = $portVersion
```

Comment (jmairboeck) [reply dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe:28]: isn't a `lib:` or `devel:` dependency always the library name?

Comment (Begasus) [reply dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe:28]: In this case there is no library involved, it only provides the headers and cmake files for detection.

File: dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+PATCH()
```
Comment (korli) [inline dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe:42]: missing newline before PATCH()

Comment (Begasus) [reply dev-cpp/gulrak-filesystem/gulrak_filesystem-1.5.12.recipe:42]: Will correct on a new release, thanks for noticing! (patch has been upstreamed aside from this (disabling -Werror)) :)

**Pull request merged**